src/modules/test/README.md

```diff
@@ -1,11 +1,17 @@
-# modulestest
-The `modulestest` application is OSConfig’s module testing framework, it allows module authors to write test recipes which in turn, drive testing of the module functionality and conformity (against the published MIM) of the Management Module.
+# ModulesTest
+The `modulestest` utility is OSConfig’s module testing framework, it allows module authors to write Test Recipes which in turn, drive testing of the module functionality and conformity (against the published MIM) of the Management Module.

```

Comment:
>The `modulestest` utility is OSConfig’s module testing framework [](http://example.com/codeflow?start=0&length=64)

How about:

"ModulesTest (binary name: modulestest) is OSConfig's functional module test utility"